sdk/identity/Azure.Identity/src/Credentials/InteractiveBrowserCredential.cs

```diff
@@ -168,6 +171,32 @@ public override AccessToken GetToken(TokenRequestContext requestContext, Cancell
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>An <see cref="AccessToken"/> which can be used to authenticate service client calls.</returns>
         public override async ValueTask<AccessToken> GetTokenAsync(TokenRequestContext requestContext, CancellationToken cancellationToken = default)
+        {
+            return await GetTokenImplAsync(true, PopTokenRequestContext.FromTokenRequestContext(requestContext), cancellationToken).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Obtains an <see cref="AccessToken"/> token for a user account silently if the user has already authenticated, otherwise the
+        /// default browser is launched to authenticate the user. Acquired tokens are cached by the credential instance. Token lifetime and
+        /// refreshing is handled automatically. Where possible, reuse credential instances to optimize cache effectiveness.
+        /// </summary>
+        /// <param name="requestContext">The details of the authentication request.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>An <see cref="AccessToken"/> which can be used to authenticate service client calls.</returns>
+        public AccessToken GetToken(PopTokenRequestContext requestContext, CancellationToken cancellationToken = default)
+        {
+            return GetTokenImplAsync(false, requestContext, cancellationToken).EnsureCompleted();
+        }
+
+        /// <summary>
+        /// Obtains an <see cref="AccessToken"/> token for a user account silently if the user has already authenticated, otherwise the
+        /// default browser is launched to authenticate the user. Acquired tokens are cached by the credential instance. Token lifetime and
+        /// refreshing is handled automatically. Where possible, reuse credential instances to optimize cache effectiveness.
+        /// </summary>
+        /// <param name="requestContext">The details of the authentication request.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>An <see cref="AccessToken"/> which can be used to authenticate service client calls.</returns>
+        public async ValueTask<AccessToken> GetTokenAsync(PopTokenRequestContext requestContext, CancellationToken cancellationToken = default)
```

Comment:
Nit: I realize these won't (or shouldn't) hurt if exposed when the feature flag isn't passed, but aren't would-be callers in a state of non-support then? It might work, but do we test it if `ISupportsProofOfPossession` isn't fully implemented and utilized?

Reply:
Good catch - this should be wrapped in the feature flag also